lib/services/presets.go

```diff
@@ -179,6 +182,7 @@ func defaultAllowRules() map[string][]types.Rule {
 			types.NewRule(types.KindConnectionDiagnostic, RW()),
 			types.NewRule(types.KindDatabase, RW()),
 			types.NewRule(types.KindLoginRule, RW()),
+			types.NewRule(types.KindDatabaseService, RO()),
```

Comment:
Should KindLicense and KindDownload be included here as well? You added them to the editor role above.

Reply:
No, removed them in the editor role.
Thank you